tests/php/Integration/Command/Export/ExportCommandTest.php

```diff
@@ -4,61 +4,46 @@
 
 namespace PhelTest\Integration\Command\Export;
 
-use Phel\Command\CommandConfig;
+use Gacela\Framework\Config;
 use Phel\Command\CommandFactory;
-use Phel\Command\CommandFactoryInterface;
-use Phel\Compiler\Analyzer\Environment\GlobalEnvironment;
-use Phel\Compiler\CompilerFactory;
-use Phel\Formatter\FormatterFactory;
-use Phel\Interop\InteropConfig;
-use Phel\Interop\InteropFactory;
-use Phel\Runtime\RuntimeFactory;
-use Phel\Runtime\RuntimeInterface;
+use Phel\Runtime\RuntimeSingleton;
 use PhelTest\Integration\Util\DirectoryUtil;
 use PHPUnit\Framework\TestCase;
 
 final class ExportCommandTest extends TestCase
 {
-    public function testExportCommandMultiple(): void
+    public static function setUpBeforeClass(): void
     {
-        $currentDir = __DIR__ . '/Fixtures/test-cmd-export-multiple/';
+        Config::setApplicationRootDir(__DIR__);
+        Config::init();
+    }
 
-        $runtime = $this->createRuntime();
-        $runtime->addPath('test-cmd-export-multiple\\', [$currentDir . 'src']);
-        $runtime->addPath('phel\\', [__DIR__ . '/../../../src/phel']);
-        $runtime->loadNs('phel\core');
+    public function setUp(): void
+    {
+        RuntimeSingleton::reset();
+    }
 
+    public function testExportCommandMultiple(): void
+    {
         $command = $this
-            ->createCommandFactory($currentDir)
-            ->createExportCommand($runtime);
+            ->createCommandFactory()
+            ->createExportCommand()
+            ->addRuntimePath('test-cmd-export-multiple\\', [__DIR__ . '/src/test-cmd-export-multiple/']);
 
         $this->expectOutputRegex('~Exported namespaces:~');
         $this->expectOutputRegex('~TestCmdExportMultiple/Adder~');
         $this->expectOutputRegex('~TestCmdExportMultiple/Multiplier~');
 
         $command->run();
 
-        self::assertFileExists("{$currentDir}/PhelGenerated/TestCmdExportMultiple/Adder.php");
-        self::assertFileExists("{$currentDir}/PhelGenerated/TestCmdExportMultiple/Multiplier.php");
+        self::assertFileExists(__DIR__ . '/PhelGenerated/TestCmdExportMultiple/Adder.php');
```

Comment:
Not sure if this works on Windows, maybe replace all those `/` by `DIRECTORY_SEPARATOR`?

Reply:
Good question... but for now I would not bother about changing this. Feel free to create a follow-up PR with this idea, but keep in mind that there might be more places (not just this file).